sspl_test/test_configuration_upgrade.py

```diff
@@ -0,0 +1,134 @@
+
+import os
+import unittest
+import sys
+
+from cortx.utils.conf_store import Conf
+
+PROJECT_ROOT = os.path.dirname(os.path.abspath(__file__)).replace(
+    "sspl_test", "low-level/framework"
+)
+sys.path.append(PROJECT_ROOT)
+from base.conf_upgrade import ConfUpgrade
+
+
+class TestConfUpgrade(unittest.TestCase):
+
+    def setUp(self):
+        base_config = "low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml"
```

Comment:
if available in sspl_constants.py use from there

Reply:
This test can be executed even is SSPL is not installed, so this base_config path is path from repo(where ever cloned locally). Hence that path will no be available in sspl_constants.

Reply:
ok